tests/E2E/Cli/phparkitect.php

```diff
@@ -13,11 +13,13 @@
     $mvc_class_set = ClassSet::fromDir(__DIR__.'/../_fixtures/mvc');
 
     $rule_1 = Rule::allClasses()
+        ->exclude('App\Controller\BaseController')
```

Comment:
If we want to keep consistency and have rules readable as english sentences `excluded` or `except` sound better to me, what do you think?

Reply:
`except` seems better in my opinion, I agree.